### DISCUSSION TITLE
Add to winget

### DISCUSSION THREAD

**Discussion by dejanstojanovic** [valinet/ExplorerPatcher#482] [Ideas]
It would be amazing if this application shows up in winget search

**Comment by valinet** [comment]
Please open non-issues in discussions.

I will look into it, idk how easy it is to maintain that tho, thanks for the suggestion.

**Reply by BraINstinct0** [reply]
I doubt if it'd be useful to be added to winget; well I sure can manage uploading that(it's strange that anyone can do it, rather than the Repo owner-but that's how it works.)
If you want automation, I'm sure you can chain a github action that adds EP to microsoft/winget-pkgs whenever a new Release is made.

... I'm not sure if this should be done though, since it'd be necessary to set MaximumOSVersion for installable conditions. Will figure out if it's possible, and if so, might be worth adding.

I will test if it's feasible in the meantime. However if 1) you abs want to have EP on WinGet and 2) also want a hassle-free method but 3) don't care too much that it's handled by someone else, [winget-pkgs-automation](https://github.com/vedantmgoyal2009/winget-pkgs-automation) does the job.

**Reply by valinet** [reply]
Yeah, look it up, personally I don’t want to deal with that, if you want to maintain and is maintainable relatively easily, without rearchitecting the whole repo, I have nothing against it. Good luck!

**Reply by BraINstinct0** [reply]
Yes, only thing I needed is this confirmation.
1) I assure you that rearchitecting is not needed. If so, I'll just withdraw WinGet support and add deprecation&redirect to it.
2) May I use the above as permission to 'add' this to winget? Just double-checking to be sure. If it is added, it means that ppl can install/update ExplorerPatcher using `winget`, nothing more.

Will report back when it is added.

**Reply by valinet** [reply]
Yeah, sure. Anyway, kind of funny anyone can add anything, but I mean, yeah, that’s how legacy download portals like cnet, Softpedia etc work anyway, I guess Microsoft is as desperate to build up a big repository fast. Then they will realize they will be in a situation similar to the Store, where “mozilla firefox” shows Firefox as the 40th result with all the unimaginable crap in between, but, oh well, “it’s business fellas”…

**Reply by BraINstinct0** [reply]
True, true. Don't see this as a 'good move', but there are users(if you know what it does, and don't want to write install scripts, and can believe full-management by third-party contributors(I def don't), `winget` can be good) so I'll add support soon™️. Won't bother you once it's setup and running(no need to, anyone who wants to install can get it from Github)
